<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>体温登记</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{
    background-color: rgb(255, 255, 255);
}
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>50</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="font">
            <font>
             <pointsize>50</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QGroupBox
{
	border: 3px solid;
	border-top-color: rgb(167, 201, 247);
	border-left-color: none;
	border-right-color: none;
	border-bottom-color: none;
}</string>
           </property>
           <property name="title">
            <string>连接扫描枪</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="topMargin">
             <number>9</number>
            </property>
            <item row="0" column="2">
             <widget class="QPushButton" name="ui_btn_connect">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>30</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
	border: 3px solid rgb(255, 255, 255);
	border-radius: 5px;
	padding: 5px;
	background: rgb(120, 174, 249);
}
QPushButton:pressed {
	border: 3px solid rgb(255, 255, 255);
	border-radius: 5px;
	padding: 5px;
	background: rgb(223, 220, 220);
}</string>
              </property>
              <property name="text">
               <string>打开</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QComboBox" name="ui_cb_com">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>30</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="ui_btn_refresh">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>30</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
	border: 3px solid rgb(255, 255, 255);
	border-radius: 5px;
	padding: 5px;
	background: rgb(120, 174, 249);
}
QPushButton:pressed {
	border: 3px solid rgb(255, 255, 255);
	border-radius: 5px;
	padding: 5px;
	background: rgb(223, 220, 220);
}</string>
              </property>
              <property name="text">
               <string>刷新</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="ui_lab_status">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#ui_lab_status{
border-image: url(:/img/is_open_off.png);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="font">
            <font>
             <pointsize>50</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QGroupBox
{
	border: 3px solid;
	border-top-color: rgb(167, 201, 247);
	border-left-color: none;
	border-right-color: none;
	border-bottom-color: none;
}</string>
           </property>
           <property name="title">
            <string>人员信息</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <property name="topMargin">
             <number>9</number>
            </property>
            <item row="0" column="0">
             <widget class="QListWidget" name="ui_lw_visitor_info">
              <property name="font">
               <font>
                <pointsize>30</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <pointsize>50</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox
{
	border: 3px solid;
	border-top-color: rgb(167, 201, 247);
	border-left-color: none;
	border-right-color: none;
	border-bottom-color: none;
}
QLineEdit{
	border: 3px solid rgb(180, 208, 246);
}</string>
         </property>
         <property name="title">
          <string>健康状况</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <item row="1" column="1">
           <widget class="QLineEdit" name="ui_lne_temperature_min">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>50</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>36.0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="ui_lne_temperature_max">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>50</pointsize>
             </font>
            </property>
            <property name="text">
             <string>37.3</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QLineEdit" name="ui_lne_temperature">
            <property name="font">
             <font>
              <pointsize>50</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>50</pointsize>
             </font>
            </property>
            <property name="text">
             <string>正常范围</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>50</pointsize>
             </font>
            </property>
            <property name="text">
             <string>当前体温</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="4">
           <widget class="QLabel" name="ui_lab_alarm">
            <property name="styleSheet">
             <string notr="true">#ui_lab_alarm{
border-image: url(:/img/health_abnormal.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="4">
           <widget class="QPushButton" name="ui_btn_OK">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>80</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>50</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	border: 3px solid rgb(255, 255, 255);
	border-radius: 5px;
	padding: 5px;
	background: rgb(120, 174, 249);
}
QPushButton:pressed {
	border: 3px solid rgb(255, 255, 255);
	border-radius: 5px;
	padding: 5px;
	background: rgb(223, 220, 220);
}</string>
            </property>
            <property name="text">
             <string>放行</string>
            </property>
            <property name="shortcut">
             <string>Return</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
